### DISCUSSION TITLE
Is possible verify if an uploaded photo have any sensible/nsfw content?

### DISCUSSION THREAD

**Discussion by HenriqueRamos13** [supabase/supabase#12421] [General]
Hi, Supabase provides any method for verify if an image have sensible content like nsfw?

**Comment by jopfre** [comment]
Nothing native in supabase I think. You would need to use an external api, or build your own api with some existing packages.

**Comment by ioucyf** [comment]
like what @jopfre said, nothing native in supabase. However,
maybe something like the following?

- https://cloud.google.com/vision/docs/detecting-safe-search
- https://www.edenai.co/technologies/image
